nixos/modules/services/x11/hardware/cmt.nix

```diff
@@ -47,7 +47,11 @@ in {
     assertions = [
       {
         assertion = !config.services.xserver.libinput.enable;
-        message = "cmt and libinput are incompatible, you cannot enable both (in services.xserver).";
+        message = ''
+          cmt and libinput are incompatible, you cannot enable both (in services.xserver).
+          plasma5 enables libinput by default. You must declare "services.xserver.libinput.enable = false"
```

Comment:
I think the message about plasma5 is irrelevant, many other desktop manager modules already enable libinput by default. How about
```
cmt and libinput are incompatible, meaning you cannot enable them both.
To use cmt you need to disable libinput with `services.xserver.libinput.enable = false`.
```

Reply:
Wouldn’t it be nice to tell the user that libinput is enabled by default somewhere? I was quite perplexed and thought it was a weird bug.

So how about
```
cmt and libinput are incompatible, meaning you cannot enable them both.
To use cmt you need to disable libinput with `services.xserver.libinput.enable = false`. If you haven’t enabled it in configuration.nix, it’s enabled by default on a 
different xserver module.
```

Reply:
I suppose that's fine too :+1: